<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="power_preset">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Balanced</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gaming</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Creative</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Custom</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_power">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Power preset</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_CPU">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU boost</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="power_CPU">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>30</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="power_GPU">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_GPU">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>GPU boost</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="power_apply">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QLabel" name="fan_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Fan preset</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="fan_preset">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Automatic</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Manual</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="fan_label_rpm">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>RPM</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="fan_apply">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>100</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QSlider" name="fan_rpm">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBothSides</enum>
    </property>
   </widget>
   <widget class="QLabel" name="fan_rpmDisplay">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>439</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
